### PR TITLE
Add advanced tx details screen

### PR DESCRIPTION
**Pull request by elgatovital**
Changes proposed in this pull request:
- Add advanced tx details screen
- Adjust nav graph

@gnosis/mobile-devs

### CODE REVIEW

File: app/src/main/java/io/gnosis/safe/ui/transactions/details/TransactionDetailsFragment.kt [hunk 1]
```diff
@@ -141,6 +142,11 @@ class TransactionDetailsFragment : BaseViewBindingFragment<FragmentTransactionDe
             }
         }
 
+        binding.advanced.setOnClickListener {
+            //TODO: pass arguments
```
Comment (jpalvarezl) [inline app/src/main/java/io/gnosis/safe/ui/transactions/details/TransactionDetailsFragment.kt:146]: This is where we wire @biafra23 changes into yours?

File: app/src/main/java/io/gnosis/safe/ui/transactions/details/view/ValueItem.kt [hunk 2]
```diff
@@ -9,7 +9,7 @@ import io.gnosis.safe.R
 import io.gnosis.safe.databinding.ViewTimeInfoItemBinding
 import timber.log.Timber
 
-class TimeInfoItem @JvmOverloads constructor(
```
Comment (jpalvarezl) [inline app/src/main/java/io/gnosis/safe/ui/transactions/details/view/ValueItem.kt:12]: I am a bit confused by this name. Is this what we use for dates as well as other stuff too? Like a label + some value?

Comment (elgatovital) [reply app/src/main/java/io/gnosis/safe/ui/transactions/details/view/ValueItem.kt:12]: You are absolutely right, that's why I have renamed it to ValueItem. Maybe LabeledValueItem would be more descriptive? The naming contest is officially open :)

File: app/src/main/java/io/gnosis/safe/ui/transactions/details/TransactionDetailsFragment.kt (hunk 1 above)
Comment (elgatovital) [reply app/src/main/java/io/gnosis/safe/ui/transactions/details/TransactionDetailsFragment.kt:146]: yes

**Pull request merged**